src/plugins/intel_gpu/src/kernel_selector/kernels/gather/gather_kernel_ref.cpp

```diff
@@ -272,10 +272,14 @@ JitConstants GatherKernelRef::GetJitConstants(const gather_params& params) const
         std::vector<std::string> idx_order;
         if (params.inputs[0].GetDims().size() == 4 && GetGatherIndexDim(params).v == 0 && !params.inputs[1].is_dynamic() &&
             params.inputs[1].LogicalSize() == 1) {
-            idx_order = idx_order = {"(f)", "(y)", "(x)", "(1)"};
+            idx_order = {"(f)", "(y)", "(x)", "(1)"};
+        } else if (params.inputs[0].GetDims().size() == 4 && GetGatherChannelIndex(params) == 1 && !params.inputs[1].is_dynamic() &&
+                   params.inputs[1].LogicalSize() == 1) {
```

Comment:
hi @michal-miotk , here your suggestion can fix customer's issue.
But when I added test case, I found a strange phenomenon.
For same model in test case, it fails in the OpenVINO function test. but if I test it via python script, it will be correct. more details in jira.